official/vision/beta/projects/yolo/modeling/backbones/darknet.py

```diff
@@ -36,36 +36,41 @@
 [2] Alexey Bochkovskiy, Chien-Yao Wang, Hong-Yuan Mark Liao
     YOLOv4: Optimal Speed and Accuracy of Object Detection. arXiv:2004.10934
 """
-import collections
 
+import collections
 import tensorflow as tf
 
 from official.vision.beta.modeling.backbones import factory
 from official.vision.beta.projects.yolo.modeling.layers import nn_blocks
 
+# builder required classes
 
-class BlockConfig(object):
-  """Get layer config to make code more readable.
 
-    Args:
-        layer: string layer name
```

Comment:
Can you specify the args as follows
layer: A `str`, for layer name.
Also use punctuation (like periods at the end).

Can you go through all args in this PR, and fix?

Reply:
Reformatted to match TF Vision